pypyr/pipelinerunner.py

```diff
@@ -124,11 +124,12 @@ def main(pipeline_name, pipeline_context_input, working_dir, log_level):
                                 string.
         working_dir: path. looks for ./pipelines and modules in this directory.
         log_level: int. Standard python log level enumerated value.
+        log_file: str. File to which log should be appended.
```

Comment:
log_file rename to `log_path`
type is "path" (i.e path-like) rather than "str". 
Rephrase description: "Append log to this path."